extras/jenkins/K3S/Jenkinsfile

```diff
@@ -69,7 +70,7 @@ pipeline {
             # Upgrade everything; we always run with latest patches.
             sudo apt -y upgrade
             # Make sure our kubeconfig dir exists…
-            mkdir ~/.kube || true
+            mkdir $HOME/.kube || true
```

Comment:
Question: For my learning, my reading on `$HOME` vs `~` is that `$HOME` has a readability advantage and maybe a slight compatibility advantage?  Is that basically correct or is there a MARA specific reason for this change?

Reply:
This was something that @dekobon brought up in an earlier review, where $HOME is preferred over ~ for the reasons you note.